slack-emoji.el

```diff
@@ -123,11 +123,7 @@ This runs asynchronously, splitting the emojis in batches of `slack-emoji-job-ba
                                          (slack-url-copy-file
                                           url
                                           path
-                                          ,team
-                                          :success (let ((e emoji))
-                                                     (lambda ()
-                                                       (funcall ',(lambda (emoji) (push-new-emoji emoji)) e))))
```

Comment:
mmm I am afraid this won't work: this is the else branch of the if above.
This means if the file is already available, add that file to the emojis that we can display otherwise download it and on success add the file to the emojis that we can display.

You are removing the second bit, which means you will only able to see files that you have already obtained.

Reply:
The problem is that the emoji is not available in lambda func for some reason. so basically let is stays outside of lambda and wont de in context of lambda

Reply:
Refactored it a bit, so now it will download if not exists and then call push-new-emoji, in that way it works good and not breaking anythiing :)